src/main/resources/assets/minestuck/models/block/glowing_purple_spike.json

```diff
@@ -0,0 +1,148 @@
+{
+  "credit": "Made with Blockbench by Dweblenod",
+  "textures": {
+    "0": "minestuck:block/glowing_purple_bricks",
```

Comment:
I kinda prever a proper texture name over a number name, even if it'd be something generic like `texture` (that is, unless there's an actual significance to calling it 0)
This would also apply to any other models that uses numbers as texture names